Vagrantfile.default

```diff
@@ -1,6 +1,6 @@
 Vagrant.configure("2") do |config|
     config.vm.box = "fredrikhgrelland/hashistack"
-    config.vm.box_version = ">= 0.7, < 0.8"
+   config.vm.box_version = ">= 0.7, < 0.8"
```

Comment:
Why do you keep another version og vagrant here?

Reply:
I m just trying different things

Reply:
I thought we concluded on friday that if you still get the same fail, then you change back to the new version!

Reply:
yes,  I will change  .